cmd/neofs-node/config/grpc/config.go

```diff
@@ -31,6 +31,20 @@ func (x *Config) Endpoint() string {
 	return v
 }
 
+// ConnectionLimit returns the value of "conn_limit" config parameter.
+//
+// Returns 0 if the value is negative.
```

Comment:
```suggestion
// Returns 0 if the value is not a positive integer.
```
e.g. `Hello, world!` -> zero too